android/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.java

```diff
@@ -117,6 +117,12 @@ public void setSubtitleStyle(SubtitleStyle style) {
             subtitleLayout.setFixedTextSize(TypedValue.COMPLEX_UNIT_SP, style.getFontSize());
         }
         subtitleLayout.setPadding(style.getPaddingLeft(), style.getPaddingTop(), style.getPaddingRight(), style.getPaddingBottom());
+        if (style.getOpacity() != 0) {
+            subtitleLayout.setAlpha(style.getOpacity());
```

Comment:
```suggestion
            subtitleLayout.setAlpha(style.getOpacity());
            subtitleLayout.setVisibility(View.VISIBLE);
```
if you change opacity from 0 to 1 it will not be visible !